### ISSUE TITLE
FATAL ERROR: Config file not found

### ISSUE THREAD

**Issue by spoom** [nrabinowitz/pjscrape#26]
It is not possible to pass any arguments to the script, cause there is an error evaluating the arguments:
phantomjs pjscrape/pjscrape.js pjs.config.js fooparm
FATAL ERROR: Config file not found: fooparm

**Comment by nrabinowitz** [comment]
That's correct, at the moment all parameters passed to Pjscrape are interpreted as config files. What are you trying to pass in?

**Comment by spoom** [comment]
The url to scrape and maxDepth.
http://pastebin.com/QjVATqDx

BTW: The site http://www.zeit.de/video/index is not working 
- Opening http://www.zeit.de/video/2012-09/1835172200001#autoplay
  ERROR: Page did not load (status=fail): http://www.zeit.de/video/2012-09/1835172200001#autoplay

I think they are detecting to much requests and then ban the IP. I think a "break" or delay would be good.

Thanks!
